charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.2.3
+version: 12.2.4
```

Comment:
this is a new feature now and should raise minor version.

Reply:
Version has been bumped to 12.3.1.